.github/workflows/build.yml

```diff
@@ -69,6 +69,69 @@ jobs:
           maven-username: ${{ secrets.NEXUS_USERNAME }}
           maven-password: ${{ secrets.NEXUS_PASSWORD }}
 
+  test_and_run_command:
```

Comment:
this job is a condition to deploy top nexus. We don't want to deploy on nexus if the tests fail